geps/gep-2648/index.md

```diff
@@ -0,0 +1,415 @@
+  everything is working - basically, as long as the targeted object exists, and
+  the modifications are valid, the metaresource is valid, and this should be
+  straightforward to communicate in one or two Conditions. The `status` stanza
+  in BackendTLSPolicy is an example of one of the recommended ways to achieve this.
+* Direct Policy Attachment SHOULD only be used to target objects in the same
+  namespace as the Policy object. Allowing cross-namespace references brings in
+  significant security concerns, and/or difficulties about merging cross-namespace
+  policy objects. Notably, Mesh use cases may need to do something like this for
+  consumer policies, but in general, Policy objects that modify the behavior of
+  things outside their own namespace should be avoided unless it uses a handshake
+  of some sort, where the things outside the namespace can opt–out of the behavior.
+  (Notably, this is the design that we used for ReferenceGrant).
+
+## Apply Policies to Sections of a Resource
+
+The `sectionName` field of `targetRef` can be used to target a specific section of other resources:
+
+* Service.Ports.Name
+* Gateway.Listners.Name
+* HTTPRoute.Rules.Name (once they are added in GEP-995, PR at https://github.com/kubernetes-sigs/gateway-api/pull/2593)
+
+For example, the RetryPolicy below applies to a RouteRule inside an HTTPRoute.
+(or rather, it will when GEP-995 merges).
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: http-app-1
+  labels:
+    app: foo
+spec:
+  hostnames:
+  - "foo.com"
+  rules:
+  - name: bar
+    matches:
+    - path:
+        type: Prefix
+        value: /bar
+    backendRefs:
+    - name: my-service1
+      port: 8080
+---
+apiVersion: networking.acme.io/v1alpha2
+kind: RetryPolicy
+metadata:
+  name: foo
+spec:
+  maxRetries: 5
+  targetRef:
+    name: http-app-1
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    sectionName: bar
+```
+
+If a `sectionName` is specified, but does not exist on the targeted object, the
+Policy must fail to attach, and the policy implementation should record a
+`resolvedRefs` or similar Condition in the Policy's status.
+
+When multiple Policies of the same type target the same object, one with a
+`sectionName` and one without, the more specific policy (i.e., the one with a
+`sectionName`) will have its entire `spec` applied to the named section.
+The less specific policy will also have its `spec` applied to the target but
+MUST not affect the named section. The less specific policy will have its `spec`
+applied to all other sections of the target that are not targeted by any other
+more specific policies.
+
+## User discoverability and status
+
+### Standard label on CRD objects
+
+Each CRD that defines a Direct Policy object MUST include a label that specifies that
+it is a Policy object, and that label MUST specify that the object is a `direct`
+one.
+
+The label is `gateway.networking.k8s.io/policy: direct`.
```

Comment:
If we have this label, why would we still need to follow a rule that place the suffix "Policy" on the name of the CRD?

Reply:
For ease of use of humans, only.